docs/concepts.rst

```diff
@@ -0,0 +1,88 @@
+ToDD Concepts
+================================
+
+Before exploring the details of how ToDD actually works, it's important to first spend some
+time understanding a few fundamental concepts.
+
+Groups
+------
+In order to solve the problems that ToDD was created to solve, it was necessary to consider scale
+from the beginning. In ToDD, testing is performed much more like "cattle" than "pets". As a result,
+tests are not performed with discrete endpoints, such as ToDD agents, but rather with "groups".
+
+.. NOTE::
+
+   At it's lowest level, testing in ToDD is done either from a group of agents, or between groups of
```

Comment:
One of those annoying things where an apostrophe does imply ownership...except when it doesn't, for the specific it's/it is case